src/recordlinker/linking/link.py

```diff
@@ -91,11 +91,30 @@ def check_and_update_score(self, earned_points, pass_lbl, rms, mmt, cmt, grade):
             self._update_score_tracking_row(earned_points, pass_lbl, rms, mmt, cmt, grade)
 
 
+def invoke(
```

Comment:
Could we rename this something like `invoke_evaluator` or `apply_comparison` or just something like that (just giving examples, I'm good with any combination that combines some application verb with a noun suggesting evaluation or comparison)? No logical problems in the code, but when looking at the function on its own outside of line 148, its context isn't super clear.

Reply:
changed to `invoke_evaluator`.